specification/eventgrid/data-plane/Microsoft.AVS/preview/2023-09-01-preview/PrivateCloud.json

```diff
@@ -0,0 +1,216 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-09-01-preview",
+    "title": "Schema of Azure VMware Solution events published to Azure Event Grid",
+    "description": "Describes the schema of the Azure VMware Solution events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+  },
+  "paths": {},
+  "definitions": {
+    "AVSPrivateCloudUpdatingEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.AVS.PrivateCloudUpdating event.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AVSPrivateCloudEventProperties"
+        }
+      ],
+      "properties": {}
+    },
+    "AVSPrivateCloudUpdatedEventData": {
```

Comment:
Why does this have both AVS and PrivateCloud? I thought we were just going to do AVS. Also, it should be cased as "Avs"

Reply:
AVS is the offering, PrivateCloud is the resource type and the subject of the event. 
I believe the agreement was to prefix every type with AVS to distinguish the from other potential cluster, privateCloud, scriptExecution, etc.

Reply:
Sounds good